test/Github/Tests/Api/CurrentUser/DeployKeysTest.php

```diff
@@ -53,10 +44,6 @@ public function shouldCreateKey()
         $this->assertEquals($expectedValue, $api->create($data));
     }
 
-    /**
-     * @test
-     * @expectedException \Github\Exception\MissingArgumentException
```

Comment:
These annotations cannot just be removed.

Reply:
Sure, I got one broken Sniff that causes thksat.

Will fix, than